tcms/core/tests/test_admin.py

```diff
@@ -140,3 +140,11 @@ def test_superuser_should_be_able_to_delete_any_user(self):
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertNotContains(response, '<a href="/admin/auth/user/%d/change/">%s</a>' %
                                (self.regular_user.pk, self.regular_user.username), html=True)
+
+    def test_admin_add_view(self):
+        self.client.login(  # nosec:B106:hardcoded_password_funcarg
+            username=self.superuser.username,
+            password='password')
+        response = self.client.get(reverse('admin:testruns_testrun_add'))
+        self.assertRedirects(response, reverse('admin:testruns_testrun_changelist'))
```

Comment:
this should go into `tcms/testruns/tests/test_admin.py` because it is related to the testurns app